pkgs/applications/misc/cherrytree/default.nix

```diff
@@ -11,21 +10,16 @@ stdenv.mkDerivation rec {
     sha256 = "0xal09ijgxbzvp003s40xbrfnpq3ald1fw8nnpqq3yg7h6g6c5pw";
   };
 
-  buildInputs = with pythonPackages;
-  [ python gettext wrapPython pygtk dbus-python pygtksourceview ];
+  nativeBuildInputs = [ gettext ];
 
-  pythonPath = with pythonPackages;
-  [ pygtk dbus-python pygtksourceview ];
+  propagatedBuildInputs = with python.pkgs; [ pygtk dbus-python pygtksourceview ];
 
   patches = [ ./subprocess.patch ];
 
   installPhase = ''
-    python setup.py install --prefix="$out"
-
-    for file in "$out"/bin/*; do
-        wrapProgram "$file" \
-            --prefix PYTHONPATH : "$(toPythonPath $out):$PYTHONPATH"
-    done
+    runHook preInstall
+    ${python.interpreter} setup.py install --prefix=$out
```

Comment:
This look pretty generic. Can't we just remove this custom installPhase?

Reply:
addressed in e167c8b23bc0b9d83a9d76f0dff53278fc1652a3